app/controllers/medicaid/contact_homeless_controller.rb

```diff
@@ -3,5 +3,20 @@
 module Medicaid
   class ContactHomelessController < StandardStepsController
     include MedicaidFlow
+
+    private
+
+    def skip?
+      mail_sent_to_residential? ||
+        reliable_mail_address?
```

Comment:
Super Minor Style: does this need to be 2 lines?

Reply:
nope! fixed at 9e16339